Vortworte, Impressum & AStA

tobkratz opened the pull request:
### Motivation for this Change :octocat:
*shrug* 
maybe some :cake:. Mostly to help the Ophasen-Team

### Things I've done
- AStA Book-n-Drive Section hinzugefügt
- Moved Comic to correct location
- `Warum gendern wir` Sektion im Impressum hinzugefügt
- Minor changes

### Tests
- [X] built with `mdbook serve` and looked up every changes

tobkratz committed:
src/_res/img/comic_self_description.png: moved to comic subfolder and fixed path in src/wilkommen/vortwort-ophase.md

tobkratz committed:
src/wilkommen/vortwort.md: gecybert, gekommentiert und den file path von src/_res/img/comics/comic_self_description.ong korrigiert

tobkratz committed:
src/wilkommen/vorwort.md: added kommentar und eventuelles TODO, das Dekanat nochmal anzuschreiben, und nach aktuallität des Vorworts zu fragen

tobkratz committed:
src/wilkommen/vortwort.md: gegendert mit :star:

tobkratz committed:
src/wilkommen/kommentierter-studenplan.md: added TODO

tobkratz committed:
src/impressum.md: added mich zur Redaktion, removed alte Daten und add oinforz Seite zum mdbook

tobkratz committed:
src/impressum.md: added Gendern Block

tobkratz committed:
src/wilkommen/vorwort-fs.md: fixed year of Fachschaftsphoto

tobkratz committed:
src/uni/asta.md: Eule hat jetzt Farbe und neue book-n-drive Sektion

Xiphoseer commented on src/willkommen/vorwort-ophase.md:
```diff
@@ -1,21 +1,22 @@
+*Wenn du es auf diese Seite geschafft hast und diesen Artikel liest<!--hier wurde mal ein bisschen gecybert-->, dann stehen die Chancen gut, dass deine Orientierungsphase gerade begonnen hat oder du bereits mitten drin bist.*
 
-Die Ophase am Fachbereich Informatik ist eine Veranstaltung mit langer Geschichte: In etwa seit es den Fachbereich Informatik gibt, gibt es auch eine Orientierungsphase für Erstsemester. Der Sinn dieser Veranstaltung ist es, dir die Universität, und den Fachbereich Informatik insbesondere, näher zu bringen, dich mit deinen Kommilitoninnen und Kommilitonen, also denjenigen Leuten, mit denen zusamm ...
+Die Ophase am Fachbereich Informatik ist eine Veranstaltung mit langer Geschichte: In etwa seit es den Fachbereich Informatik gibt, gibt es auch eine Orientierungsphase für Erstsemester. Der Sinn dieser Veranstaltung ist es, dir die Universität, und den Fachbereich Informatik insbesondere, näher zu bringen, dich mit deinen Kommiliton*innen<!-- Einheitliches gendern im Gesamten Artiekl-->, also de ...
```
Muss leider ein `\*` sein zum gendern, sonst wird es kursiv wenn der Parser einen zweiten Stern findet

Xiphoseer commented on src/willkommen/vorwort-ophase.md:
```diff
@@ -1,21 +1,22 @@
 <br>
 
-![xkcd.org](../_res/img/comic_self_description.png)
+![xkcd.org](../_res/img/comics/comic_self_description.png) <!--Diesmal wirklich der richtige Path :D-->
```
Würdest du das `comic_` noch rausnehmen, das hätte ich nur nur eingefügt weil ich den Unterordnen nicht von Anfang an verwendet habe

Xiphoseer commented on src/willkommen/vorwort.md:
```diff
@@ -1,5 +1,6 @@
 # Vorwort des Dekans
-
+<!-- altes Vorword -->
```
Vorwort

Xiphoseer commented on src/willkommen/vorwort-ophase.md:
```diff
@@ -1,21 +1,22 @@
 # Willkommen zur Ophase
 
-*Wenn du dieses Heft in der Hand hältst und diesen Artikel liest, dann stehen die Chancen gut, dass deine Orientierungsphase gerade begonnen hat oder du bereits mitten drin bist.*
+*Wenn du es auf diese Seite geschafft hast und diesen Artikel liest<!--hier wurde mal ein bisschen gecybert-->, dann stehen die Chancen gut, dass deine Orientierungsphase gerade begonnen hat oder du bereits mitten drin bist.*
```
An sich spricht nichts gegen in-line comments, aber ich würde es soweit wie möglich vermeiden, da die (muss ich allerdings noch verifizieren) weil sie HTML sind auch im generierten Buch noch drin sind.

Xiphoseer reviewed (commented):
Super, vielen Dank für diesen Beitrag. Hab nur ein paar kleine Anmerkungen.

tobkratz committed:
changes requested by @Xiphoseer

added changes requested by @Xiphoseer at https://github.com/d120/cyberinforz/pull/2

tobkratz committed:
fixed typo

Xiphoseer commented:
Jetzt müsste nur noch die Datei mit dem Comic umbenannt werden 🙈

miterion commented on src/impressum.md:
```diff
@@ -6,17 +6,20 @@ Die Redaktion tagt derzeit unregelmäßig. Die Termine werden über die offene M
 Namentlich gekennzeichnete und anonyme Beiträge geben nicht notwendigerweise die Meinung der Redaktion wieder. Alle Rechte, insbesondere das der Verfilmung, vorbehalten.
 
+**Warum gendern wir?**
+<https://genderdings.de/gender/gendern/>
```
Sollten wir hier nicht eher auf die TU Richtlinien verweisen?

tobkratz replied:
Wäre auch ne Idee, ich finde die nur umfangreicher.

Xiphoseer replied:
Ich würde das erstmal so machen wie von Tobi vorgeschlagen und dann ggf die Ophasen-Leitung fragen was sie besser finden

tobkratz commented:
> Jetzt müsste nur noch die Datei mit dem Comic umbenannt werden

Da verstehe ich nicht, was du meinst :confused:

Xiphoseer commented:
> Da verstehe ich nicht, was du meinst :confused:

Ah, ich hatte im ersten Review gebeten den Dateinamen des Comics noch zu verändern, jetzt wo er verschoben ist, jetzt habe ich gerade gesehen, du hast den auskommentiert, hat das einen besondern Grund?

Xiphoseer commented on src/willkommen/vorwort-ophase.md:
```diff
@@ -1,21 +1,22 @@
+Die Ophase am Fachbereich Informatik ist eine Veranstaltung mit langer Geschichte: In etwa seit es den Fachbereich Informatik gibt, gibt es auch eine Orientierungsphase für Erstsemester. Der Sinn dieser Veranstaltung ist es, dir die Universität, und den Fachbereich Informatik insbesondere, näher zu bringen, dich mit deinen Kommiliton\*innen, also denjenigen Leuten, mit denen zusammen du dein Stud ...
 
-All das, was dir diese Woche geboten wird, ist von Studierenden organisiert und durchgeführt, um Studienanfängerinnen und -anfängern wie dir einen optimalen Start ins Studium zu ermöglichen. Die Tutor*innen, die dich diese Woche betreuen, sind Studierende, die, genau wie du jetzt, auch einmal eine Ophase mitgemacht haben und sich ein paar Semester später dazu entschlossen haben, das, was sie dama ...
+All das, was dir diese Woche geboten wird, ist von Studierenden organisiert und durchgeführt, um Studienanfänger*innen<!-- Einheitliches gendern im Gesamten Artiekl--> wie dir einen optimalen Start ins Studium zu ermöglichen. Die Tutorinnen und Tutoren, die dich diese Woche betreuen, sind Studierende, die, genau wie du jetzt, auch einmal eine Ophase mitgemacht haben und sich ein paar Semester spä ...
```
```suggestion
All das, was dir diese Woche geboten wird, ist von Studierenden organisiert und durchgeführt, um Studienanfänger\*innen wie dir einen optimalen Start ins Studium zu ermöglichen. Die Tutor\*innen, die dich diese Woche betreuen, sind Studierende, die, genau wie du jetzt, auch einmal eine Ophase mitgemacht haben und sich ein paar Semester später dazu entschlossen haben, das, was sie damals gelernt haben, an neue Studierende wie dich weiterzugeben.
<!-- Einheitliches gendern im gesamten Artikel-->
```

tobkratz committed:
Update src/willkommen/vorwort-ophase.md

Co-authored-by: Xiphoseer <Xiphoseer@users.noreply.github.com>

tobkratz committed:
corrected file name 3rd time

tobkratz committed:
src/_res/img/comics/self_description.png: corrected file name

Xiphoseer reviewed (approved):
Danke!
